packages/server/modules/workspaces/utils/isUserLastWorkspaceAdmin.ts

```diff
@@ -0,0 +1,13 @@
+import { WorkspaceAcl } from '@/modules/workspaces/domain/types'
+
+export const isUserLastWorkspaceAdmin = (
```

Comment:
This is missing unit tests. I know the functionality is somewhat tested in the services, but in these cases i think it is required to fully test all the edge cases of this functionality.